.neuro/live.yml

```diff
@@ -0,0 +1,106 @@
+kind: live
+
+defaults:
+  preset: gpu-k80-small-p
+  life_span: 1d
+
+volumes:
+  data:
+    remote: storage:$[[ flow.project_id ]]/data
+    mount: /project/data
+    local: data
+  code:
+    remote: storage:$[[ flow.project_id ]]/code
+    mount: /project/code
+    local: code
+  config:
+    remote: storage:$[[ flow.project_id ]]/config
+    mount: /project/config
+    local: config
+    read_only: True
+  notebooks:
+    remote: storage:$[[ flow.project_id ]]/notebooks
+    mount: /project/notebooks
+    local: notebooks
+  results:
+    remote: storage:$[[ flow.project_id ]]/results
+    mount: /project/results
+    local: results
+  project:
+    remote: storage:$[[ flow.project_id ]]
+    mount: /project
+    local: .
+
+images:
+  myimage:
+    ref: image:$[[ flow.project_id ]]
+    dockerfile: $[[ flow.workspace ]]/Dockerfile
```

Comment:
Seems like we've lost the Dockerfile in this repo

Reply:
Wow, good catch, added it, thanks!